src/VoyagerServiceProvider.php

```diff
@@ -95,14 +95,11 @@ public function boot(Router $router, Dispatcher $event)
 
         $this->loadViewsFrom(__DIR__.'/../resources/views', 'voyager');
 
-        if (app()->version() >= 5.4) {
-            $router->aliasMiddleware('admin.user', VoyagerAdminMiddleware::class);
+        $router->aliasMiddleware('admin.user', VoyagerAdminMiddleware::class);
+        $this->loadMigrationsFrom(realpath(__DIR__.'/../publishable/database/migrations'));
```

Comment:
If migrations are no longer publishable, we should probably move them out of that directory.  We can handle that in a subsequent PR.  No need to change it now, just wanted it documented.